### DISCUSSION TITLE
HAOS with Q35 and VIRT GL GPU

### DISCUSSION THREAD

**Discussion by Stephan-4711** [tteck/Proxmox#3514] [General]
Hi,

I'm trying to use most hardware rescources as possible also in view to LLMs on HAOS.
I created a new VM because my old install wasn't based on q35.
When I switch from VirtIO-GPU tp VirtGL-GPU the machine doesn't start at all. 
There is no error message or anything else. Can anybody give me some help`?

**Comment by cscashby** [comment]
I have a similar question - would love to be able to do this - Intel Alder Lake
